commons/src/main/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManagerConfig.java

```diff
@@ -32,6 +32,9 @@ public class UniversalRemoteStorageManagerConfig extends AbstractConfig {
     private static final String OBJECT_STORAGE_FACTORY_CONFIG = OBJECT_STORAGE_PREFIX + "factory";
     private static final String OBJECT_STORAGE_FACTORY_DOC = "The factory of an object storage implementation";
 
+    private static final String OBJECT_STORAGE_KEY_PREFIX_CONFIG = OBJECT_STORAGE_PREFIX + "prefix";
+    private static final String OBJECT_STORAGE_KEY_PREFIX_DOC = "The object storage path prefix";
+
```

Comment:
I think this should go without `OBJECT_STORAGE_PREFIX`? Everything that has `OBJECT_STORAGE_PREFIX` is meant for passing subsequently to the factory instance for initialization, but the prefix config (oh the pun!) is probably a URSM's parameter?